### PR TITLE
Add MLOps Docker Image

### PR DESCRIPTION
**Pull request by eedorenko**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 pool:
   vmImage: 'Ubuntu 16.04'
 
-container: dtzar/conda3mlops:latest
+container:
+  image: mlopscr.azurecr.io/public/mlops/mlopspython:latest
```
Comment (dtzar) [inline azure-pipelines.yml]: Can't we use the public image?

File: environment_setup/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,13 @@
 FROM conda/miniconda3
 
-COPY . /setup
+LABEL org.label-schema.vendor = "Microsoft" \
+      org.label-schema.url = "dockerhub" \
```
Comment (dtzar) [inline environment_setup/Dockerfile]: This should be URL to docker hub image

Comment (eedorenko) [reply environment_setup/Dockerfile]: Yes. Done.

File: azure-pipelines.yml (hunk 1 above)
Comment (eedorenko) [reply azure-pipelines.yml]: Not yet. Until it is published on Docker Hub we are using this one.

**Pull request merged**